### DISCUSSION TITLE
What invention lets you look right through a wall?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1120] [Galaxy Trivia] [answered]
**Bonus:** What is black and white and red all over?

*answer with emojis*

**Comment by LeleDallas** [comment, marked as answer]
A window


> **Bonus:** What is black and white and red all over?
> 
> _answer with emojis_

📰🗞️

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/LeleDallas?achievement=galaxy-brain&tab=achievements) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by peterbaccaro** [comment]
The answer to the riddle regarding the invention that lets you look right through a wall is **a window**.

What is black and white and red all over?
📰 **Newspaper**
🌞 🐧 **Sunburnt Penguin**
